locales/tr/tr.json

```diff
@@ -1,618 +1,618 @@
+    "Manage and log out your active sessions on other browsers and devices.": "Yönetmek ve diğer tarayıcılar ve cihazlar üzerinde etkin oturumları oturum açın.",
     "Manage and logout your active sessions on other browsers and devices.": "Diğer tarayıcılarda ve cihazlarda aktif oturumlarınızı yönetin ve oturumu kapatın.",
     "Manage API Tokens": "API Jetonlarını Yönetin",
     "Manage Role": "Rolü Yönet",
     "Manage Team": "Takımı Yönet",
-    "March": "March",
-    "Marshall Islands": "Marshall Islands",
-    "Martinique": "Martinique",
-    "Mauritania": "Mauritania",
+    "March": "Mart",
+    "Marshall Islands": "Marshall Adaları",
+    "Martinique": "Martinik",
+    "Mauritania": "Moritanya",
     "Mauritius": "Mauritius",
-    "May": "May",
-    "Mayotte": "Mayotte",
-    "Mexico": "Mexico",
-    "Micronesia, Federated States Of": "Micronesia",
+    "May": "-ebilmek",
+    "Mayotte": "Mayotte. kgm",
+    "Mexico": "Meksikalı",
+    "Micronesia, Federated States Of": "Mikronezya",
     "Moldova": "Moldova",
-    "Monaco": "Monaco",
-    "Mongolia": "Mongolia",
-    "Montenegro": "Montenegro",
-    "Month To Date": "Month To Date",
+    "Monaco": "Monako",
+    "Mongolia": "Moğolistan",
+    "Montenegro": "Karadağ",
+    "Month To Date": "Bugüne Kadar Ay",
     "Montserrat": "Montserrat",
-    "Morocco": "Morocco",
-    "Mozambique": "Mozambique",
+    "Morocco": "Fas",
+    "Mozambique": "Mozambik",
     "Myanmar": "Myanmar",
     "Name": "Ad",
-    "Namibia": "Namibia",
+    "Namibia": "Namibya",
     "Nauru": "Nauru",
     "Nepal": "Nepal",
-    "Netherlands": "Netherlands",
+    "Netherlands": "Hollanda",
     "Nevermind": "Boşver",
-    "New": "New",
-    "New :resource": "New :resource",
-    "New Caledonia": "New Caledonia",
+    "New": "Yeni",
+    "New :resource": "Yeni :resource",
+    "New Caledonia": "Yeni Kaledonya",
     "New Password": "Yeni Şifre",
-    "New Zealand": "New Zealand",
-    "Next": "Next",
-    "Nicaragua": "Nicaragua",
-    "Niger": "Niger",
-    "Nigeria": "Nigeria",
+    "New Zealand": "Yeni Zelanda",
+    "Next": "Sonraki",
+    "Nicaragua": "Nikaragua",
+    "Niger": "Nijer",
+    "Nigeria": "Nijerya",
     "Niue": "Niue",
-    "No": "No",
-    "No :resource matched the given criteria.": "No :resource matched the given criteria.",
-    "No additional information...": "No additional information...",
-    "No Current Data": "No Current Data",
-    "No Data": "No Data",
-    "no file selected": "no file selected",
-    "No Increase": "No Increase",
-    "No Prior Data": "No Prior Data",
-    "No Results Found.": "No Results Found.",
-    "Norfolk Island": "Norfolk Island",
-    "Northern Mariana Islands": "Northern Mariana Islands",
-    "Norway": "Norway",
+    "No": "Hayır",
+    "No :resource matched the given criteria.": "Hayır :resource verilen kriterlerle eşleşti.",
```

Comment:
`"No :resource matched the given criteria.": "Bu kriterlere uyan bir :resource yok.",`